src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -72,6 +163,26 @@ void shouldThrowAnExceptionWhenNameIsNull() throws Exception {
         result.andExpect(status().isBadRequest());
     }
 
+    @Test
+    void itShouldReturnBadRequestWhenIdIsNull() throws Exception {
+        //GIVEN
+        Instant timeCreated = Instant.now();
+
+        UpdateWalletDTO updateWalletDTO = new UpdateWalletDTO(null, "@#$%^&");
+        given(walletService.updateWallet(updateWalletDTO))
+                .willReturn(new WalletDTO(null, "", timeCreated));
+
+        //WHEN
```

Comment:
Why you wrote it in upper case?

Reply:
TO have better overview and orientation, in case if test method will be too long? Is that upper case an issue? Please let me know whether it should be upper or lower case convention.

Reply:
changed to lower case.